packages/fuchsia_remote_debug_protocol/lib/src/fuchsia_remote_connection.dart

```diff
@@ -13,9 +13,11 @@ import 'common/network.dart';
 import 'dart/dart_vm.dart';
 import 'runners/ssh_command_runner.dart';
 
-final String _ipv4Loopback = InternetAddress.LOOPBACK_IP_V4.address; // ignore: deprecated_member_use
+final String _ipv4Loopback =
+    InternetAddress.LOOPBACK_IP_V4.address; // ignore: deprecated_member_use
```

Comment:
Why not use the non-deprecated value?

Reply:
This is just from the auto-formatter.  I can change it, though.